apps/arweave/src/ar_packing_server.erl

```diff
@@ -414,6 +427,85 @@ record_buffer_size_metric() ->
 			ok
 	end.
 
+record_packing_benchmarks(PackingStateRef, MaxRate, PackingRate, Schedulers) ->
+	prometheus_gauge:set(packing_latency_benchmark,
+		[protocol, pack, spora_2_5], ?PACKING_LATENCY_MS),
```

Comment:
spora_2_5 packing involves a different [number of RX hashes](https://github.com/ArweaveTeam/arweave/blob/master/apps/arweave/include/ar_consensus.hrl#L2). The latency does not linearly depend on the number of RX programs so we cannot simply prorate it unfortunately, without making experiments. The spora_2_5 packing is not involved in the new protocol, although some miners are still repacking their legacy storages, so may be we should just exclude spora_2_5 from these metrics.